### PR TITLE
Bump gratia-probe.spec release number

### PR DESCRIPTION
**Pull request by mwestphall**

### CODE REVIEW

File: rpm/gratia-probe.spec [hunk 1]
```diff
@@ -607,6 +607,9 @@ This is a transitional dummy package for gratia-probe-slurm; it may safely be re
 %files slurm
 
 %changelog
+* Tue Dec 10 2024 Matt Westphall <westphall@wisc.edu.> -  2.8.5-1
+- Log gratia probe failures to aid debugging (SOFTWARE-6037)
+
```
Comment (brianhlin) [inline rpm/gratia-probe.spec]: There's also the fix from https://opensciencegrid.atlassian.net/browse/SOFTWARE-5558

**Review by brianhlin — APPROVED** [review]
One missing changelog entry but otherwise LGTM. Pre-approving

File: rpm/gratia-probe.spec (hunk 1 above)
Comment (mwestphall) [inline rpm/gratia-probe.spec]:
```suggestion
  - Fix NameError in post-install script (SOFTWARE-5558)
```

File: rpm/gratia-probe.spec [hunk 2]
```diff
@@ -607,6 +607,9 @@ This is a transitional dummy package for gratia-probe-slurm; it may safely be re
 %files slurm
 
 %changelog
+* Tue Dec 10 2024 Matt Westphall <westphall@wisc.edu.> -  2.8.5-1
+- Log gratia probe failures to aid debugging (SOFTWARE-6037)
+  - Fix NameError in post-install script (SOFTWARE-5558)
```
Comment (mwestphall) [inline rpm/gratia-probe.spec]:
```suggestion
- Fix NameError in post-install script (SOFTWARE-5558)
```

**Pull request merged**